src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -153,7 +155,8 @@ public static function create(
       $configuration['third_party_settings'],
       $container->get('current_user'),
       $container->get('entity_type.manager'),
-      $container->get('twig')
+      $container->get('twig'),
+      $container->get('config.factory')
```

Comment:
You can always call the parent::create and then add your particular container injection